### PR TITLE
dealbot

### PR DESCRIPTION
**Pull request by coryschwartz**

### CODE REVIEW

File: charts/dealbot/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        command: [ "controller" ]
+        args: [ "--configpath", "/config.toml" ]
+        env: []
+        volumeMounts:
+        - name: config-volume
+          mountPath: /config.toml
+          subPath: config.toml
+          readOnly: true
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
```
Comment (willscott) [inline charts/dealbot/templates/deployment.yaml:53]: https://github.com/filecoin-project/dealbot/pull/37 will allow settings to be specified either directly as args or as env vars, and removes need for this

**Review by willscott — COMMENTED** [review]
looks good. other hook-ups to expect short term:
* the controller will want to talk to postgres
* we may eventaully want a public port inbound to the controller.

File: charts/dealbot/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: dealbot-worker
+        chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+    spec:
+      volumes:
```
Comment (ognots) [inline charts/dealbot/templates/deployment.yaml:82]: we'll need some helm values here  to specify the shared volume (EFS) required between Lotus and Dealbot. I assume it's the workers that need access

File: charts/dealbot/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Filecoin Dealbot
+
+This helm chart deploys the filecoin dealbot
+
+## Dependencies
+
+### Lotus
+
+A [lotus](https://github.com/filecoin-project/lotus) daemon must be running with its' API exposed for Visor to consume.
```
Comment (ognots) [inline charts/dealbot/README.md:9]: s/visor/deal-bot/g

File: charts/dealbot/templates/deployment.yaml (hunk 2 above)
Comment (willscott) [reply charts/dealbot/templates/deployment.yaml:82]: correct :+1:

**Review by hannahhoward — COMMENTED** [review]
We don't actually need prometheus for the dealbot worker FWIW -- there is no metrics endpoint there.

**Pull request merged**